### ISSUE TITLE
RFE: Admirals beyond the first one give +2 trade route capacity

### ISSUE THREAD

**Issue by whorble45** [Infixo/Civ6-Mods#16]
Currently with the mod, some great admirals when recruited seem to give +2 trade route capacity even if they are not the first one to be recruited. I didn't notice if every admiral had this problem, but I distinctly remember it happening when one of them was recruited. By the end of the game I had ~60 trade routes but only ~30 cities, which I can't see happening any other way. Is there more info I could provide?

**Comment by Infixo** [comment]
The modifier responsible for granting TRs can only be executed once, regardless on how many GAs are recruited. This is how vanilla game works (see below).
The mod doesn't change that. I also specifically tested that scenario to make sure that it only had granted 2 TRs even when more GAs had been recruited. If you have such an effect then I surely need a save file and logs to inspect the situation.

```
<ModifierId>ELIZABETH_TRADE_ROUTES_MODIFIER</ModifierId>
<ModifierType>MODIFIER_PLAYER_ADJUST_TRADE_ROUTE_CAPACITY</ModifierType>
<OwnerRequirementSetId>PLAYER_HAS_GREAT_ADMIRAL_REQUIREMENTS</OwnerRequirementSetId>
<RunOnce>true</RunOnce>
<Permanent>true</Permanent>
```

**Comment by whorble45** [comment]
Sorry, I jumped to conclusions here. I tried to reproduce it after a reinstall with a smaller set of mods and this time it worked as intended. I'll try to be more thorough next time!

(Off topic: I've been having all kinds of weird issues lately, like e.g. Gorgo's cities somehow getting faith from population, despite installing no mods that would do that? I suspect some database corruption? Could errors in one mod cascade to cause completely unrelated issues? I'm thinking index mismatches or something like that.)
